features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/UserDisabledScenario.swift

```diff
@@ -16,7 +16,6 @@ internal class UserDisabledScenario: Scenario {
     }
 
     override func run() {
-        Bugsnag.setUser(nil, withEmail: nil, andName: nil)
```

Comment:
Would it be better to change expectation on this scenario that all fields are null, but then add an extra scenario for default behaviour?

Reply:
Added an extra scenario which verifies the default user information only contains an ID.